batch/batch/resource_usage.py

```diff
@@ -82,6 +112,58 @@ def memory_usage_bytes(self) -> Optional[int]:
                 return int(f.read().rstrip())
         return None
 
+    def overlay_storage_usage_bytes(self) -> int:
+        return shutil.disk_usage(self.container_overlay).used
+
+    def io_storage_usage_bytes(self) -> int:
+        if self.io_volume_mount is not None:
+            return shutil.disk_usage(self.io_volume_mount).used
+        return 0
+
+    def is_attached_disk(self) -> bool:
+        return self.io_volume_mount is not None and os.path.ismount(self.io_volume_mount)
+
+    async def network_bandwidth(self) -> Tuple[Optional[float], Optional[float]]:
+        now_time_msecs = time_msecs()
+
+        iptables_output, _ = await check_shell_output(
+            f'''
+iptables -t mangle -L -v -n -x | grep "{self.veth_host}" | awk '{{ if ($7 == "{self.veth_host}") print $2 }}'
+'''
+        )
+        now_download_bytes = int(iptables_output.decode('utf-8').rstrip())
+
+        iptables_output, _ = await check_shell_output(
+            f'''
+iptables -t mangle -L -v -n -x | grep "{self.veth_host}" | awk '{{ if ($6 == "{self.veth_host}") print $2 }}'
+'''
```

Comment:
Could we possibly get both values from the same exec? Especially considering both iptables calls are the same it would be nice to do this in 1.